tests/lxd-vm/spread.yaml

```diff
@@ -0,0 +1,15 @@
+project: spread
+
+backends:
+    lxd-vm:
+        systems:
+            - ubuntu-20.04
```

Comment:
This should probably also test 16.04, 18.04 and 22.04 as per #182 (assuming there are VM images for all old releases as well)

Reply:
16.04 and 18.04 don't work in a VM, lxd-agent does not come start, so I've limited this to 20.04 and 22.04 for now. Another issue I ran into, is that tests need to be run one by one, otherwise the host runs out of disk space.

Reply:
Eh, and 22.04 will only work when https://github.com/snapcore/spread/pull/184 or other similar fix lands.